src/uct/api/v2/uct_v2.h

```diff
@@ -223,6 +262,21 @@ uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
 ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
                                  const uct_md_mem_dereg_params_t *params);
 
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Get ep's attributes.
+ *
+ * This routine should be called only if the endpoint was created with the
+ * UCT_EP_PARAM_FIELD_SOCKADDR or UCT_EP_PARAM_FIELD_CONN_REQUEST flags.
```

Comment:
this is documentation for "local_address" and "remote_address" fields. in the future we will want to extend uct_ep_query to return also non-sockaddr related information

Reply:
@yosefe Fixed